src/analysis/retail/monk/windwalker/modules/talents/Serenity.tsx

```diff
@@ -12,10 +12,14 @@ import { STATISTIC_ORDER } from 'parser/ui/StatisticBox';
 import { ABILITIES_AFFECTED_BY_DAMAGE_INCREASES } from '../../constants';
 import { TALENTS_MONK } from 'common/TALENTS';
 
-const DAMAGE_MULTIPLIER = 0.2;
+const DAMAGE_MULTIPLIER = 0.15;
 
 const MOD_RATE = 2;
-const MOD_RATE_ABILITIES = [SPELLS.FISTS_OF_FURY_CAST.id, TALENTS_MONK.RISING_SUN_KICK_TALENT.id];
+const MOD_RATE_ABILITIES = [
+  SPELLS.FISTS_OF_FURY_CAST.id,
+  TALENTS_MONK.RISING_SUN_KICK_TALENT.id,
+  TALENTS_MONK.STRIKE_OF_THE_WINDLORD_TALENT.id,
```

Comment:
Are you missing any other affected abilities here? At a glance I notice Rushing Jade Wind costs 1 chi and has a cooldown

Reply:
Rushing Jade Wind 🤮

I double checked on wowhead: https://www.wowhead.com/spell=152173/serenity, that's all that seems to be missing.